app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Accordion.kt

```diff
@@ -0,0 +1,145 @@
+package com.hedvig.android.design.system.hedvig
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.design.system.hedvig.AccordionDefaults.Size
+import com.hedvig.android.design.system.hedvig.tokens.AccordionTokens
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.TextAccordion
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.TextPrimary
+
+@Composable
+fun AccordionList(items: List<AccordionData>, modifier: Modifier = Modifier, size: Size = Size.Small) {
+  Column(modifier) {
+    for ((index, perilItem) in items.withIndex()) {
+      var isExpanded by rememberSaveable { mutableStateOf(false) }
+      AccordionItem(
+        isExpanded = isExpanded,
+        onClick = {
+          isExpanded = !isExpanded
+        },
+        title = perilItem.title,
+        description = perilItem.description,
+        size = size,
+      )
+      if (index != items.lastIndex) {
+        Spacer(Modifier.height(4.dp))
+      }
```

Comment:
More of a stylistic change, but I've found it much more convenient lately to just do the flipped of this, like we do here
https://github.com/HedvigInsurance/android/blob/0ec4aaee4da0c6552868e06c462ab30c2536de5b/app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments/discounts/DiscountRow.kt#L39-L41

I like it for these two reasons
* The divider is at the start of the foreach, so it's easy to know it's there and does not get pushed further down when there is a lot of code in-between.
* The check is always done when `!= 0`, so no need to reference the list itself

No need to change it here, just thought I'd mention it :D

Reply:
With that said, especially in this case where it is not a divider which we want to add in-between items, but it is only a spacer, this is usually much easier to do with doing spacedBy on the Column instead https://github.com/HedvigInsurance/android/blob/a042b5c5562b532699424ab27894b0f7534be519/app/audio-player-ui/src/main/kotlin/com/hedvig/android/audio/player/internal/FailedAudioPlayerCard.kt#L46
That way there's not even any need for extra code, index checks etc. And you can turn the .withIndex() into just a normal `for (perilItem in items) {` instead.